common/azure-common-test/src/main/java/com/azure/common/test/TestMode.java

```diff
@@ -6,12 +6,19 @@
  * The possible testing modes.
  */
 public enum TestMode {
+
     /**
      * Record data from a live test.
      */
     RECORD,
+
     /**
      * Playback data from an existing test session.
      */
     PLAYBACK,
+
+    /**
+     * Default to None if no mode
+     */
+    NONE,
```

Comment:
What does a test mode of NONE mean? Is it a valid state to be in?

Reply:
Added this as some sdk tests depend on this.

Reply:
Can you clarify what you mean? I'm not sure how tests could depend on this if you just added it? Do you have a printer to where this is used?